lib/tmuxinator/project.rb

```diff
@@ -184,7 +184,7 @@ def tmux_has_session?(name)
       # Please see issue #564.
       unescaped_name = name.shellsplit.join("")
 
-      !!(sessions =~ /^#{unescaped_name}:/)
+      !(sessions !~ /^#{unescaped_name}:/)
```

Comment:
Weird that Rubocop now prefers for this to be phrased as a double negative, I'd thought `!!` was a well-established Ruby idiom. I'd probably rather drop the `!`s entirely and use the return value of `=~` (first matching index or nil). Not a huge deal, though.